<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>操作系统课程设计</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-11</y>
      <width>621</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>619</width>
          <height>378</height>
         </rect>
        </property>
        <attribute name="label">
         <string>任务一：3个并发进程</string>
        </attribute>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>241</width>
           <height>371</height>
          </rect>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>0</y>
           <width>351</width>
           <height>181</height>
          </rect>
         </property>
         <property name="title">
          <string>CPU利用率</string>
         </property>
         <widget class="QProgressBar" name="progressBar_6">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>90</y>
            <width>261</width>
            <height>23</height>
           </rect>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>259</x>
           <y>190</y>
           <width>351</width>
           <height>181</height>
          </rect>
         </property>
         <property name="title">
          <string>当前时间</string>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>70</y>
            <width>251</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>619</width>
          <height>378</height>
         </rect>
        </property>
        <attribute name="label">
         <string>任务二：添加系统调用</string>
        </attribute>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>260</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>选择源文件</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>260</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>选择目标文件</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>260</y>
           <width>99</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>拷贝</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar_7">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>120</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>116</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>619</width>
          <height>378</height>
         </rect>
        </property>
        <attribute name="label">
         <string>任务三：添加设备驱动</string>
        </attribute>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>66</y>
           <width>291</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>150</y>
           <width>291</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Waiting...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>输入：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>160</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>myDevice：</string>
         </property>
        </widget>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>260</y>
           <width>176</width>
           <height>27</height>
          </rect>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>619</width>
          <height>378</height>
         </rect>
        </property>
        <attribute name="label">
         <string>任务四：系统监视器</string>
        </attribute>
        <widget class="QTabWidget" name="tabWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>621</width>
           <height>381</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>进程</string>
          </attribute>
          <widget class="QTableWidget" name="tableWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>621</width>
             <height>341</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>资源</string>
          </attribute>
          <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>611</width>
             <height>111</height>
            </rect>
           </property>
           <property name="title">
            <string>CPU利用率</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <widget class="QProgressBar" name="progressBar">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>40</y>
              <width>211</width>
              <height>21</height>
             </rect>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
           <widget class="QProgressBar" name="progressBar_2">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>80</y>
              <width>211</width>
              <height>21</height>
             </rect>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
           <widget class="QProgressBar" name="progressBar_3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>80</y>
              <width>211</width>
              <height>21</height>
             </rect>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
           <widget class="QProgressBar" name="progressBar_4">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>40</y>
              <width>211</width>
              <height>21</height>
             </rect>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>40</y>
              <width>41</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>CPU0</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>80</y>
              <width>41</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>CPU2</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>540</x>
              <y>40</y>
              <width>41</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>CPU1</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>540</x>
              <y>80</y>
              <width>41</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>CPU3</string>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>110</y>
             <width>611</width>
             <height>101</height>
            </rect>
           </property>
           <property name="title">
            <string>内存利用率</string>
           </property>
           <widget class="QProgressBar" name="progressBar_5">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>60</y>
              <width>251</width>
              <height>23</height>
             </rect>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>60</y>
              <width>201</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>230</y>
             <width>611</width>
             <height>121</height>
            </rect>
           </property>
           <property name="title">
            <string>网络</string>
           </property>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>40</y>
              <width>71</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>正在接收</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>40</y>
              <width>71</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>正在发送</string>
            </property>
           </widget>
           <widget class="QLabel" name="downloadRate">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>40</y>
              <width>101</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>0.0B/s</string>
            </property>
           </widget>
           <widget class="QLabel" name="uploadRate">
            <property name="geometry">
             <rect>
              <x>410</x>
              <y>40</y>
              <width>91</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>0.0B/s</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>80</y>
              <width>81</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>总计已接收</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>80</y>
              <width>81</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>总结已发送</string>
            </property>
           </widget>
           <widget class="QLabel" name="download">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>80</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>0字节</string>
            </property>
           </widget>
           <widget class="QLabel" name="upload">
            <property name="geometry">
             <rect>
              <x>410</x>
              <y>80</y>
              <width>111</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>0字节</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>619</width>
          <height>378</height>
         </rect>
        </property>
        <attribute name="label">
         <string>任务五：模拟文件系统</string>
        </attribute>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>161</width>
           <height>381</height>
          </rect>
         </property>
         <column>
          <property name="text">
           <string>文件树</string>
          </property>
         </column>
        </widget>
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>0</y>
           <width>461</width>
           <height>381</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
